composer.json

```diff
@@ -25,10 +25,7 @@
     "autoload": {
         "psr-4": {
             "Spatie\\TwitterStreamingApi\\": "src"
-        },
-        "classmap": [
```

Comment:
Oh right, forgot to mention that change. Right now running `composer install` in this package (not inside a project) ends with an error because it can't find that file